### DISCUSSION TITLE
Output format

### DISCUSSION THREAD

**Discussion by divdax** [pestphp/pest#729] [Q&A]
After updating to v2 i got this "beautiful" output. Are you experiencing any line breaks as well? Do you have any thoughts on what could be causing this issue?

<img width="723" alt="image" src="https://user-images.githubusercontent.com/1456739/226453965-429482f4-52d5-4b2d-b049-15d665d70c29.png">

- iTerm 3.4.19
- ohmyzsh
- ZSH_THEME="robbyrussell"

**Comment by fabio-ivona** [comment]
@divdax , @EnterpriseDNA could you share more info about your environment?

**Comment by fabio-ivona** [comment]
What's your OS?

**Comment by divdax** [comment]
For me the output in PhpStorm seems ok. iTerm is pretty much default. Just changed colors/font. Switching to another font has no effect.

**Comment by fabio-ivona** [comment]
@divdax sorry, can't reproduce your issue, could you try to change your terminal font?

**Comment by fabio-ivona** [comment]
@divdax could you share a screenshot of your iTerm settings?

this tab :point_down: 

![image](https://user-images.githubusercontent.com/8792274/226689135-aac67608-51fc-4969-b099-cce0b5adf6ae.png)

**Comment by dansysanalyst** [comment]
I've been trying to reproduce the issue as well, but no success.

@divdax , @EnterpriseDNA, out of curiosity: Do you continue experiencing the issue if you drop to another shell? For example `sh`.

<img width="696" alt="CleanShot 2023-03-21 at 18 17 23@2x" src="https://user-images.githubusercontent.com/79267265/226690264-2380a823-2a14-4e7c-b64d-4f14b3326b4b.png">

**Comment by divdax** [comment]
@fabio-ivona this are my settings. Changing the font has no effect. (changed to Monaco, and some other monotype fonts)

<img width="762" alt="image" src="https://user-images.githubusercontent.com/1456739/226717198-70e6f1ec-c339-4781-a7e6-7646194b6dec.png">

When i run the tests in the terminal of PhpStorm it looks good. Same shell zsh:

<img width="917" alt="image" src="https://user-images.githubusercontent.com/1456739/226722416-bbc1b07d-c9a8-499e-8a9e-d797bba081df.png">

Even when i change my shell to /bin/bash - same result.

**Comment by dansysanalyst** [comment]
@divdax Would you consider sharing your profile settings to help us to reproduce the issue? 

You may export it as JSON in `other actions`.

 <img src="https://user-images.githubusercontent.com/79267265/226753162-208cd5ab-b046-42df-a3e0-b5da683dbe4e.png" alt="alt text" width="600">

**Comment by fylzero** [comment]
Having this same problem using ZSH with Solarized Dark theme - I switched my color scheme to the Smoooooth theme and it looks fine now.
